includes/modules/pages/page/header_php.php

```diff
@@ -20,21 +21,20 @@
 
 $ezpage_id = (int)($_GET['id'] ?? '0');
 if ($ezpage_id === 0) {
-  zen_redirect(zen_href_link(FILENAME_DEFAULT));
+    zen_redirect(zen_href_link(FILENAME_DEFAULT));
 }
 
 $chapter_id = isset($_GET['chapter']) ? (int)$_GET['chapter'] : 0;
 $chapter_link = isset($_GET['chapter']) ? (int)$_GET['chapter'] : 0;
 
-
 if (!$sniffer->table_exists(TABLE_EZPAGES_CONTENT)) {
-  return; // early exit; db not upgraded
+    return; // early exit; db not upgraded
 }
```

Comment:
Sorry to be a PITA on this, but this section should be removed.  If that table doesn't exist, the site/site-support should be made aware of the issue instead of silently letting it go by.

Reply:
I added this a long time ago when the tables were split because people were getting in trouble trying things before doing the DB upgrade.  I don't have strong feelings but it would be easier just to let it be - it's not causing problems.